dev-db/mydumper/mydumper-0.10.3.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A high-performance multi-threaded backup (and restore) toolset for MySQL"
+HOMEPAGE="https://github.com/maxbube/mydumper"
+SRC_URI="https://github.com/maxbube/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+COMMON_DEPEND="dev-db/mysql-connector-c:=
+	dev-libs/glib:=
+	dev-libs/libpcre:=
+	dev-libs/openssl:0=
+	sys-libs/zlib:="
+DEPEND="${COMMON_DEPEND}
+	virtual/pkgconfig
+	doc? ( dev-python/sphinx )"
```

Comment:
You have actually set a python program without provide PYTHON_COMPAT=( python3_{7..9} ) and ${PYTHON_DEPS}

Reply:
Also, IIRC, you have to inherit python-any-r1 to, but probably @thesamesam or @juippis will know better than me

Reply:
No it's fine, if you don't have a BDEPEND-executable sphinx, you've got bigger problems in your system.

Reply:
To an extent, although you're going to have to add the machinery if you ever need a custom theme. Sphinx itself should be okay though, I suppose.